internal/plugin/connectors/ssh/proxy_service.go

```diff
@@ -148,7 +149,7 @@ func (proxy *proxyService) Start() error {
 			}()
 
 			go func() {
-				if err := proxy.handleConnections(chans); err != nil {
+				if err := proxy.handleConnections(chans); err != nil && err != io.EOF {
```

Comment:
Nice. This is to resiliently handle the client shutting down the connection?

Reply:
@doodlesbykumbi this looks like it's from a different PR and somehow got mixed in here?

Reply:
took it out. yeah it should be another PR